common/changes/@microsoft/api-extractor-model/type-members_2021-11-03-08-38.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor-model",
+      "comment": "Support documenting members of type aliases",
+      "type": "patch"
```

Comment:
Doesn't this add new functionality and thus this should be `minor` (same for the other two changelog files)? Can we also clarify in the change comment that this only supports documenting members of a very simple shape of type alias?